docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/node-requirements-for-rancher-managed-clusters.md

```diff
@@ -116,6 +116,8 @@ For a breakdown of the port requirements for etcd nodes, controlplane nodes, and
 
 Details on which ports are used in each situation are found under [Downstream Cluster Port Requirements](../../../getting-started/installation-and-upgrade/installation-requirements/port-requirements.md#downstream-kubernetes-cluster-nodes).
 
+**DO NOT** register a node that has the same hostname / IP as an existing node. For both node driver and custom clusters, RKE1 may prevent the node from joining and cause provisioning to get stuck. See [Nodes](https://rke.docs.rancher.com/config-options/nodes) for steps on how to override the hostname of an RKE node using the `hostname_override` option.
```

Comment:
```suggestion
:::note
You should never register a node with the same hostname or IP address as an existing node. RKE may prevent the node from joining and cause provisioning to hang. This can occur for both node driver and custom clusters. See  the [RKE documentation](https://rke.docs.rancher.com/config-options/nodes#overriding-the-hostname) to learn how to override the hostname of an RKE node using the `hostname_override` option.
:::
```

Reply:
Same questions as on the "outdated" comment

Reply:
Same questions as on the "outdated" comment

Reply:
Ah, I see what happened. We do need to make updates both to the 2.7 page and to the page in /docs, as the mirroring process is currently manual. We recently added the 2.7 directory and have been working on automatically syncing both directories whenever one updates.

Reply:
I think either `caution` or `danger` would be a better fit than `note` given the context.